### PR TITLE
avoid running the check if cluster name is empty

### PR DESCRIPTION
**Pull request by mfpierre**
### What does this PR do?

Oversight of the cluster check migration, the orchestrator check should not run if the cluster name is empty

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Describe your test plan

Write there any instructions and details that should be tested during the QA.

### CODE REVIEW

File: pkg/collector/corechecks/cluster/orchestrator/orchestrator.go [hunk 1]
```diff
@@ -147,8 +147,9 @@ func (o *OrchestratorCheck) Configure(config, initConfig integration.Data, sourc
 
 	// check if cluster name is set
 	hostname, _ := coreutil.GetHostname()
-	if clusterName := clustername.GetClusterName(hostname); clusterName != "" {
-		o.orchestratorConfig.KubeClusterName = clusterName
+	o.orchestratorConfig.KubeClusterName := clustername.GetClusterName(hostname)
+	if o.orchestratorConfig.KubeClusterName == "" {
```
Comment (nammn) [inline pkg/collector/corechecks/cluster/orchestrator/orchestrator.go:151]: Do we have some way to run unit tests for core check runs?

**Review by nammn — APPROVED** [review]
LGTM! One question regarding testing

File: pkg/collector/corechecks/cluster/orchestrator/orchestrator.go (hunk 1 above)
Comment (mfpierre) [reply pkg/collector/corechecks/cluster/orchestrator/orchestrator.go:151]: I don't think there is any specific tooling around this yet, I tested the PR locally

```
2021-04-23 15:15:51 UTC | CLUSTER | ERROR | (pkg/collector/corechecks/loader.go:72 in Load) | core.loader: could not configure check orchestrator: orchestrator check is configured but the cluster name is empty
```

Comment (nammn) [reply pkg/collector/corechecks/cluster/orchestrator/orchestrator.go:151]: 👌 I thought we might have an easy way to add tests. If that not the case it is probably not worth it to invest that time 👍

**Pull request merged**